mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -0,0 +1,60 @@
+/* © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). */
+
+if (typeof window.jQuery === 'undefined') { throw new Error('Requires jQuery'); }
```

Comment:
Yes, but it's implicit. I mean, you use jQuery when you call the function at the end, and if it is not defined the browser will throw that error by itself, so no need to do it manually.